digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -85,11 +85,11 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService, implici
     }
 
     def extractLinkId(massTransitStop: PersistedMassTransitStop): (String, Option[Long]) = {
-      "link_id" -> Some(massTransitStop.linkId)
+      if(Some(massTransitStop.linkId).contains(0)) "link_id" -> None else "link_id" -> Some(massTransitStop.linkId)
```

Comment:
Why we need to put the linkId value inside a "Some()" and the use ".contains" function to compare values when we can do it like this:
```scala
if (massTransitStop.linkId == 0) "link_id" -> None else "link_id" -> Some(massTransitStop.linkId)

Reply:
I think we can use the val isMassServicePoint and pass it as argument to this when doing the map.

```scala
    def extractLinkId(massTransitStop: PersistedMassTransitStop, isMassServicePoint: Boolean): (String, Option[Long]) = {
      "link_id" -> (if(isMassServicePoint) None else Some(massTransitStop.linkId))
    }
```

Reply:
@marquesfm Check modification on this code.
This part was rollbacked and added a if on the Map construction